research/picai/fl_nnunet/nnunet_utils.py

```diff
@@ -179,6 +180,12 @@ def __iter__(self) -> DataLoader:  # type: ignore
         # mypy gets angry that the return type is different
         return self
 
+    def shutdown(self) -> None:
```

Comment:
Some elaboration on what is involved in the shutdown of different augmenters may be helpful